### ISSUE TITLE
TO DO

### ISSUE THREAD

**Issue by VivaAB** [epfml/ml-project-1-bytemeifyoucan#4]
before monday 23/10:
SATURDAY:
- [ ] have report template with rough plan that we can follow
- [ ] tryout the dataset and understand it
- [ ] clean very basically data (remove nan basically)
- [ ] split data
- [x] run some non-optimized model on data
- [x] obtain confusion matrices
- [x] implement helper functions from ml_course

SUNDAY:
- [ ] CASIMIR: cleaning the data
- [ ] VIVA: work on functions
- [ ] YANN: implement first basic run notebook (data - model - create submission + get accuracy/f1)
- [x] check what to do for logistic with y appartient 0,1 ou -1,1

--------------------------------
- [ ] find best lambda with cross validation (try multiple folds)
- [ ] tryout sgd and gd with multiple learning rates (CV? multiple folds?)
- [ ] visualization of the data: 
1. [ ] pca to know which features are good at explainig variance 
2. [ ] visualize how model performs(loss, cross validation, test loss)
3. [ ] confusion matrices plotted? maybe plot sensitivity/recall/precision/specificity
4. [ ] which ones are wrongly classified -> multiple subplots with various features/PCs? manually look at the subjects and compare?

before friday 27/10: note: Casimir stops on Friday on this project (midterm)
- [ ] finish code and most runs
- [ ] have final model
- [ ] are we underfitting or overfitting in any way? look at train vs test errors

before Sunday 29/10:
- [ ] know how to submit project
- [ ] be almost done with report, all figures in, only need to write in a fancy way

Monday 30/10:
- [ ] proof read code
- [ ] proof read report
- [ ] submit
